src/planwise/component/scenarios.clj

```diff
@@ -27,20 +28,43 @@
   [changeset]
   (apply +' (map :investment changeset)))
 
+(defn- create-text-description
+  [changeset]
+  (let [sites         (count changeset)
+        capacity      (apply +' (mapv #(-> % (read-string) :capacity) changeset))
+        text-default  ["Create replace site." "Increase overall capacity in replace."]]
```

Comment:
Use https://clojuredocs.org/clojure.core/format.
Should be simple to use a `(cond)` over the sites count.